### ISSUE TITLE
Few Artist's all song download is capped to 48 songs

### ISSUE THREAD

**Issue by adimrsingh** [monuyadav016/Saavn-Downloader#16]
Artist page URL : https://www.saavn.com/s/artist/kishore-kumar-albums/yMoPyl3ZzyY_

This happens :

Downloading all Artist songs
Total Songs of the Artist: 12193
Total requests: 1220
Getting Song page: 0
Downloading 1. Bheegi Bheegi Raaton Mein.m4a [*] 4.5 MB / 4.5 MB @ 7.8 MB/s [##################] [100%, 0s left]    

..............

Downloading 48. Zindagi Ek Safar.m4a [*] 4.0 MB / 4.0 MB @ 1,013 kB/s [##################] [100%, 0s left]    
error() missing 1 required positional argument: 'msg'
No songs found for the artist
DONE

I don't know why, can you give this a look?

**Comment by monuyadav016** [comment]
It seems the error happened while adding meta data. Will try fixing the add_tags or will probably skip adding meta data to those songs so that the download can continue.

**Comment by monuyadav016** [comment]
@adimrsingh Now it should continue to download all songs but those not available in 320kbps will not have any meta data. Hope this is a satisfying enough fix. I'll look into it sometime later. But I can't make any guarantees.

**Comment by adimrsingh** [comment]
Yup, that did fixed the download issue. And It's a fair enough fix. I just added a commit to the download_manager.py , no functionality added just numbered the filename also I sent you a pull request regarding it.

**Comment by accountForIssues** [comment]
@monuyadav016 I also get the same error as the OP (with the current repo)

Artist Link: https://www.jiosaavn.com/artist/mahendra-kapoor-/mzleuBtK1zU_

Relevant log in my case:

```
Getting Song page: 25
...
Downloading Shri Ramdev Peer Ra Shay Pasta.m4a
 [*] 68.5 MB / 68.5 MB @ 2.1 MB/s [##################] [100%, 0s left]     
 [*] 68.5 MB / 68.5 MB @ 2.6 MB/s [##################] [100%, 0s left]     
 [*] 68.5 MB / 68.5 MB @ 1.5 MB/s [##################] [100%, 0s left]     
 [*] 68.5 MB / 68.5 MB @ 1.7 MB/s [##################] [100%, 0s left]     
error() missing 1 required positional argument: 'msg'
No songs found for the artist
DONE
```
On running it again, the file above got downloaded successfully. After a couple more files, the error happened again:

```
Getting Song page: 26
...
Downloading Var Dhagala Lagali Kal (From 'Bot Lavin Tithe Gudgulya').m4a
 [*] 12.6 MB / 12.6 MB @ 853 kB/s [##################] [100%, 0s left]     
 [*] 12.6 MB / 12.6 MB @ 783 kB/s [##################] [100%, 0s left]    
 [*] 12.6 MB / 12.6 MB @ 1.3 MB/s [##################] [100%, 0s left]      
 [*] 12.6 MB / 12.6 MB @ 935 kB/s [##################] [100%, 0s left]     
error() missing 1 required positional argument: 'msg'
No songs found for the artist
DONE
```

**Comment by monuyadav016** [comment]
> @monuyadav016 I also get the same error as the OP (with the current repo)
> ```

This error is not the same as OP's. You got an error possibly due to connectivity issues.
